dao-contracts/tests/steps/bid_escrow.rs

```diff
@@ -27,7 +27,7 @@ fn post_job_offer(
     dos_fee: Balance,
 ) {
     let timeframe = helpers::to_seconds(timeframe, time_unit);
-    w.post_offer(job_poster, timeframe, maximum_budget, dos_fee);
+    let _ = w.post_offer(job_poster, timeframe, maximum_budget, dos_fee);
```

Comment:
I changed `post_offer()` and it returns a `Result`, and here I'm ignoring the result. I must return a `Result` because in a case where the DOS fee is too low, it should fail, but not panic.
Btw. it initially was returning PostJobId or smth, but I think it doesn't make sense because we don't need it in this step.